File: first.cpp
#include<iostream>
#include<string>
using namespace std;
int main(){
    int n,r;
    cin>>n>>r;
    for (int i=0;i<n;i++){
        int f;
        string arr[n+2];
        cin>>f;
        if (f<r){
            arr[i]="Bad boi";
        }
        else
        arr[i]="Good boi";
    }
    for (int j=0;j<n;j++)[
        cout<<arr[j]<<endl;
    ]
}

File: first.py
n,r =map(int,input().split(" "))
l=[]
c1="Bad boi"
c2="Good boi"
for i in range(n):
    o=int(input())
    if o<r:
        l.append(c1)
    else:
        l.append(c2)
for j in l:
    print(j)
